### DISCUSSION TITLE
Hello everyone!

### DISCUSSION THREAD

**Discussion by VincentAraneo** [SuperTux/supertux#2761] [General]
I'm Vincent, and I'm really a fan of supertux. I love the development that the community and developers do. What I do in the community is just create levels, whether they're private or public. I only ask that you please treat me well and without insults, as I don't like to be rudely messed with. I just wanted to tell you this precentation and that I'm new here in this part of the community, and a fan since in the Milestone 1. Thanks for listening!

**Comment by MatusGuy** [comment]
huh?

**Comment by Frostwithasideofsalt** [comment]
...

**Reply by VincentAraneo** [reply]
In fact, I've already changed and I'm already a good person

**Reply by Frostwithasideofsalt** [reply]
> In fact, I've already changed and I'm already a good person

Sure, i bet a lot changed a lot since i talked to you in that YouTube comment section, and you _definitely_ just had your 13th birthday to make it so your legally allowed to go on github.

**Reply by tylerandari13** [reply]
The videos are still there. You never took then down when requested.

**Reply by VincentAraneo** [reply]
Look Now I Removed Industrial Island v2 Videos
